packages/snjs/lib/services/api/session_manager.ts

```diff
@@ -508,29 +442,8 @@ export class SNSessionManager extends PureService<SessionEvent> {
       );
       return signInResponse;
     } else {
-      if (signInResponse.error.payload?.mfa_key) {
-        if (mfaCode) {
-          await this.alertService.alert(SignInStrings.IncorrectMfa);
-        }
-        /** Prompt for MFA code and try again */
-        const inputtedCode = await this.promptForMfaValue();
-        if (!inputtedCode) {
-          /** User dismissed window without input */
-          return this.apiService.createErrorResponse(
-            SignInStrings.SignInCanceledMissingMfa,
-            StatusCode.CanceledMfa
-          );
-        }
-        return this.bypassChecksAndSignInWithRootKey(
-          email,
-          rootKey,
-          signInResponse.error.payload.mfa_key,
-          inputtedCode
-        );
-      } else {
-        /** Some other error, return to caller */
-        return signInResponse;
-      }
+      // return error to caller
```

Comment:
Please follow SNJS comment convention: `/** Sentence case */`

Reply:
Double asterisk or triple slash comments are conventionally used for documentation. E.g. [[1]](https://en.wikipedia.org/wiki/Javadoc#Structure_of_a_Javadoc_comment), [[2]](https://jsdoc.app/about-getting-started.html).

Are we consciously breaking that convention?